Solita.LocalizationEditor.UI/Properties/AssemblyInfo.cs

```diff
@@ -6,7 +6,7 @@
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
 [assembly: AssemblyTitle("Solita.LocalizationEditor")]
-[assembly: AssemblyDescription("Localization Editor for EPiServer CMS")]
+[assembly: AssemblyDescription("LocalizationResult Editor for EPiServer CMS")]

```

Comment:
Very good point, no it wasn't. Accidentally performed replace string for this one and didn't notice it...